LAKMobile/src/screens/AddJob.tsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import { useState } from "react";
+import { Button, PermissionsAndroid, ScrollView, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import { Picker } from "@react-native-picker/picker";
+import SingleLineTextInput from '../components/SingleLineTextInput';
+import MultilineTextInput from '../components/MultilineTextInput';
+import AppText from '../components/AppText';
+import {launchCamera, launchImageLibrary} from 'react-native-image-picker';
+import {MediaType} from 'react-native-image-picker';
+import Icon  from 'react-native-vector-icons/MaterialCommunityIcons';
+import { PrimaryButton, ScreenHeader } from '../components';
+import PickerItem from '../components/PickerItem';
+
+const LOCATIONS = ["-- Select a district --", "Bumthang", "Chhukha", "Dagana", "Gasa", "Haa",
+  "Lhuentse", "Mongar", "Paro", "Pema Gatshel", "Punakha", "Samdrup Jongkhar",
+  "Samtse", "Sarpang", "Thimphu", "Trashigang", "Trashi Yangtse", "Trongsa",
+  "Tsirang", "Wangdue Phodrang", "Zhemgang"];
+
+export default function AddJob() {
+  const [pickupDistrict, setPickupDistrict] = useState('Unknown'); 
+  const [dropoffDistrict, setDropoffDistrict] = useState('Unknown');
+  const [addJobInformation, setAddJobInformation] = useState({
+    jobTitle: "",
+    clientName: "",
+    deliveryDate: "",
+    packageDescription: "",
+    packageQuantity: "",
+    estimatedPrice: "",
+    pickUpLocation: "",
+    dropOffLocation: "",
+    phoneNumber: "",
+  });
+
+    const addJob = () => {
+        console.log("temp ");
+    }
+
+    const requestCameraAccess = async () => {
```

Comment:
The image upload functionality does not seem to work well on my end, not sure if that is supposed to be the case.

Gives the error on the left when you press either of the three buttons in the popup.

![image](https://user-images.githubusercontent.com/56745367/156124240-91d9d3c7-930d-4130-b33e-5e1e3226e2eb.png)

Reply:
image upload functionality to be implemented later